apollo-runtime/src/main/java/com/apollographql/apollo/internal/interceptor/ApolloAutoPersistedQueryInterceptor.java

```diff
@@ -68,13 +70,13 @@ public void interceptAsync(@NotNull final InterceptorRequest request, @NotNull f
           if (isPersistedQueryNotFound(response.errors())) {
             logger.w("GraphQL server couldn't find Automatic Persisted Query for operation name: "
                 + request.operation.name().name() + " id: " + request.operation.operationId());
-            return Optional.of(request.toBuilder().sendQueryDocument(true).build());
+            return Optional.of(request.toBuilder().sendQueryDocument(true).useHttpGetMethodForQueries(false).build());
```

Comment:
I think this should fallback to whatever user configured  in `com.apollographql.apollo.ApolloClient.Builder#useHttpGetMethodForQueries` rather than always `false`?

Reply:
Yes, it will fallback in later Interceptor chains.

According to `com.apollographql.apollo.internal.RealApolloCall.java` line. 355
'prepareInterceptorChain', the `com.apollographql.apollo.ApolloClient.Builder#useHttpGetMethodForQueries` is passing to `ApolloServerInterceptor` instead of `ApolloAutoPersistedQueryIntercaptor`.

Under this consideration,
Should we also pass useHttpGetMethodForQueries to ApolloAutoPersistedQueryIntercaptor?

Reply:
Please rebase as `useHttpGetMethodForQueries ` is now passed inside the `InterceptorRequest`. So I assume we should fall back to the original `request.useHttpGetMethodForQueries`

Reply:
Got it & updated, now it will respect to the old `request.useHttpGetMethodForQueries` for retrying as normal request after APQs is failed.

Reply:
May i clarify the original usage of 
`ApolloClient#useHttpGetMethodForQueries` 
Is it for all requests to use GETMethod regardless APQs or not? for all Queries right?

Just wanna make sure the useHttpGetMethodForPersistedQueries doesn't duplicate the function.